src/compiler/types.d.ts

```diff
@@ -1,3 +1,5 @@
+import {TreeProto} from '@ampproject/bento-compiler/dist/ast';
```

Comment:
We shouldn't be importing from `dist/`, we should be exporting into the module and importing directly from it

Reply:
I agree. Will fix this in the next version of bento-compiler